packages/checkout/widgets-lib/src/widgets/bridge/views/ClaimWithdrawal.tsx

```diff
@@ -0,0 +1,248 @@
+  const { eventTargetState: { eventTarget } } = useContext(EventTargetContext);
+
+  const { page } = useAnalytics();
+
+  useEffect(() => {
+    page({
+      userJourney: UserJourney.BRIDGE,
+      screen: 'ClaimWithdrawal',
+    });
+  }, []);
+
+  const [txProcessing, setTxProcessing] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [hasWithdrawError, setHasWithdrawError] = useState(false);
+  const [withdrawalResponse, setWithdrawalResponse] = useState<FlowRateWithdrawResponse | null>();
+
+  const goBack = useCallback(() => {
+    viewDispatch({
+      payload: {
+        type: ViewActions.GO_BACK,
+      },
+    });
+  }, [viewDispatch]);
+
+  useEffect(() => {
+    const getWithdrawalTxn = async () => {
+      if (!tokenBridge) return;
+      // get withdrawal transaction from the token bridge by receipient address and index
+      setLoading(true);
+      try {
+        const flowRateWithdrawTxnResponse = await tokenBridge?.getFlowRateWithdrawTx({
+          recipient: transaction.details.to_address,
+          index: 0, // TODO: update index from transaction when it comes through from backend
+        });
+        setWithdrawalResponse(flowRateWithdrawTxnResponse);
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.log(err);
+      } finally {
+        setLoading(false);
+      }
+    };
+    getWithdrawalTxn();
+  }, [tokenBridge]);
+
+  const handleWithdrawalClaimClick = useCallback(async () => {
+    if (!checkout || !tokenBridge || !from?.web3Provider || !withdrawalResponse) return;
+
+    if (!withdrawalResponse.pendingWithdrawal.canWithdraw || !withdrawalResponse.unsignedTx) {
+      // eslint-disable-next-line max-len, no-console
+      console.log(`Unable to process withdrawal transaction as it is not ready yet. Delay timeout at ${withdrawalResponse.pendingWithdrawal.timeoutEnd} `);
+      return;
+    }
+
+    let providerToUse = from?.web3Provider;
+    const l1ChainId = getL1ChainId(checkout.config);
+
+    setTxProcessing(true);
+
+    if (isPassportProvider(from?.web3Provider)) {
+      // user should switch to MetaMask
+      try {
+        const createProviderResult = await checkout.createProvider({ walletProviderName: WalletProviderName.METAMASK });
+        const connectResult = await checkout.connect({
+          provider: createProviderResult.provider,
+          requestWalletPermissions: true,
+        });
+        providerToUse = connectResult.provider;
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.log(err);
+        return;
+      }
+    }
+
+    let ethGasCostWei: BigNumber | null;
+    try {
+      const gasEstimate = await providerToUse.estimateGas(withdrawalResponse.unsignedTx);
+      const feeData = await providerToUse.getFeeData();
+      let gasPriceInWei: BigNumber | null;
+      if (feeData.lastBaseFeePerGas && feeData.maxFeePerGas && feeData.maxPriorityFeePerGas) {
```

Comment:
just remove `feeData.maxFeePerGas && ` from the if

Reply:
oh yes, forgot to note that